src/api/app/models/workflow/step.rb

```diff
@@ -88,6 +92,14 @@ def target_package_name(short_commit_sha: false)
 
   private
 
+  def multibuild?
+    Package.get_by_project_and_name(target_project_name, target_package_name, follow_multibuild: true).multibuild?
+  end
+
+  def multibuild_flavors
+    Package.get_by_project_and_name(target_project_name, target_package_name, follow_multibuild: true).multibuild_flavors.collect { |_flavor| "#{target_package_name}:flavor" }
```

Comment:
We need `flavor`, so it shouldn't be prefixed with `_`.

```suggestion
    Package.get_by_project_and_name(target_project_name, target_package_name, follow_multibuild: true).multibuild_flavors.collect { |flavor| "#{target_package_name}:flavor" }
```